GruntFile.js

```diff
@@ -377,16 +344,14 @@ module.exports = function (grunt) {
 
     grunt.registerTask('default', [
         'jasmine:main:build',
-        'jshint:force',
-        'jscs:force',
-        'if-missing:esri_slurp:dev',
+        'eslint:main',
```

Comment:
There is [no force option for eslint](https://github.com/sindresorhus/grunt-eslint/pull/55#issuecomment-77816277). If you really want it, there is the grunt-force-task.